img/pages/activities/svg/icon-connect.svg

```diff
@@ -1,404 +1,25 @@
-RsEtIcs/ErAk89iiA5YOHst5xbTnC1gibw/1ip85YGnyWI6ZdxjeUoeWx+YLWGIes4Ys/0jAkuKk
-xQIClnNHwakClv5R8EUFLGfZkcMHLHGVR2vI8o8ELPG8WEOWqT8RsNRtZIt/8A8ELFW/Zc766g8E
-LBEWW8jyjwQsUYTSFrL8IwFLzTdud1/FHQ6OGUKX9vYifi3qYcy5g5gwluAwZmreIKYaoQwIY6bm
-DWLas6po/bthg5hoLIFhzNS8QUyUIxQYxpw7iInGEhjGnDuIGaEJY84dxMQVXoPCmHMHMSNxijCm
-LYi5GYmjSGW7Nuhao5SReByetOTJdIQAxHZZ/ugNmp1/ZSXCRtV/GfgX/c0XoyxXiHKiCF9E9LT5
-Gklg2CibjDYHESZaAlWxnSspk2rvbdIbDjrKv9EN9Oj2uHl1UI1uRNUftOEHm9EEdIlpAzS8SqL4
-aBu62UbNwH+3f6MPcmRK+t86+v9phIkyWvfQv7cdeHARYbKMmC8wLJvnWD7PSVEmyxWkPDziuXy+
-KBUFeMLCxIjwMc8VmUIxj1rJFtm8JOQZsVDgeJErwpO8IDJF+F2RY0WpgFrKFwQetSoKRYYtStFb
-1FVEHQ7+u/0XvhzCh68olwUIrlgsiiIvFRgh+neUZaLH0YcnJtpFo7uIiHxWKuQBBFrjBbYQFSXU
-BXjASXkYARv9IcDw0PU8w3CcWOAZBh4wWV6C0TAM/IVuowcCL8IvGKYI4+WjQh7GwUMbQoHlGBYg
-OIBgUKOiWBCBHJWIG0bICgI0yxaKHBCoQGjF2dtrdyt5PotGD//wjAjUjTZng+EARmCBmBxMBSAj
-wsC4BRh3kRVEkQEYQYQfARXgJQck46NsAdEOfgF/RDwmLsvnodl8gec5IJ0L4C3CF7M8sAxMoyAI
-PC+5Yfi8GzV6hrrMwUQVgHGg30BkeOoifTPC89kiB+NnBYYvMMBkNDAEgrhhXL1isyLPIYgiMCVL
-IAhXyAK/WtogUIQTs2K+CAMDCKAeS2gFeBQxJycAcQFM7QkQkSlKElqABAjneCoRFwyBUWaCcTEu
-CcbJ/m4I5yzdRTg2CysQfpQXhbzEwGzDAxERHOjNcmKUzWcR1RhYlnn4KZ5q4A/oLMty+SJPAHD2
-tRJxwRRA1nAFgOGhLZbnYTxuGFgLeTSxIM4kgEOTJjIAwXO8AE/4aEHKipJQhNeAioEpYrMcYgWO
-hTkTCq73IDMKNgjWBdKMAAEKwEsMCEImz3F5FwiSb8BLQFYB4CQedRQGYhUzLgjncCsRFwxhioOl
-bTPyGuHFrCBBh0FesEVG4KNFBrAhqAJsHRxsFD8RHvYTFq0rCW0orOiGgSVSRMzC5hHvCEK0KGQl
-EQnpYl4SJeAEN0Qxm5fgASPA7lOAtYkXmh2GZYRsIS9GcQeAa1hoUgIiwYsCUA2xPSvA2i5GXcNw
-QbxFBNjMGERHmP18oUhoRRCyRdsi0DvgeuHqPXAHwKB51GcakwCtF0aC3ZSH1p34XWQMnow3mLL1
-K7LKkY/mDgaTaKJ1vd9ufXZG8uW/I9miZVxpGzejbtxs9G+sTtx2I1I0kYze3mAI24P/JkXErYCA
-VChkJUZCexKTl9g8SeFwwzgVDpBuWeg+fGRESRJYt8IB481KEsy1KMDGWiTISiTFnDBOieuGcCsc
-LhiC/J8JhrC/umGc+yss1CyHpFuRLxQloeDeGVkJBLNVdhH2V1YSsrBbFPFKQpuGuxEWCCGh/ksS
-Bxuwa3t1Q7i3VxcMQe1gxUK2IIimBkUDQyKdC8ZFOphcEDqwFlhOgt2SMGpgRNh7ge2AC9Hsk0iH
-WNHgQyQxXa0I6ImuLLMSgXYuCALtnDAklpoFhrAnuWFcC8UF4VZNgM7ZIt6sQSZIhYJbNQFdGCYB
-JgB+wYOMd6kmLgCCauKEIakmLhiXaiLATHMSPGB50Lkkl9LAClKWRRu+BPo97KcE3YQFarOIW0QO
-uIpzqzdoPUvZPM+aks6FBovCgrEBu3UTN4RbN3HBkOY4UDAj3QQWa5YvwJzBdokUGIJqwopiluWR
-wiUVwGLJu2HQEoENQ1dFeLdq4oZwqyYuGF0zcHXApVQ4R0HQTFhJzMIcwFPAwUvERopZYB8QBqA5
-MFb8zhcEzUSFscyISzVxdcBNxqC5+F/WTNho9T+vcrDRQ4LfIyIBcxfRFIAYEcR8FPZRFgt/wC4I
-oC8A33JgzQNa0HdBKImsG6YZaZGJKQQQk8GUOf1fpkwHO4LQzLFgU/NFaFiCbSYPJhCoTyAbEIWg
-LyyIYhg+aPQiiGZgbkEC+eGGYXkOGR3QfRE2KtBhBKAPiBnYtCQG0LoBOLBFgSEL0Aq0iZeVAwSW
-AcgXRoRewXAZ1IajGzBzaD2gDSMPG3fUNRYnACDhJNAOYSUBbh5UGUIbAohZpD5CCyyxG04I0lhc
-MC6CODvipmngzLxF3slMxwYw3X9qOXqsQNAmQToi6oHWIIqwd8OkoH1K4vISfvCjrlJTrSi6Yf5r
-lyBwGM9ZPVYSMJSIyMFzSPVAw8+zoFUxfAEYgmWRFuKCQRoF0pgYgctzggStwBaAfsJKHMeDtHJD
-uEj4FnHDgC4Dw2OFvADGBXJhOXuCFD5YHsUChwQk9MQ1HhfEWwQZGtAsTKVQhHYFQisiLHbQOQSe
-5wGG0BM3hHs8LhgXVVw9cVE2eH5mXoiOrdSxazbLatCjNujiqEQmE4nHzzof8qXS6fVlJfIx7vwl
-RzuDwXDSmcgjeBP9UOTxZKjI0fHn8G/0BH6ig8fjtdN65P8BB9iOrw==
-	]]>
-</i:pgf>
-</svg>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>

```

Comment:
I basically made these SVGs by finding non-bloated versions of them in other pages and just find/replacing the colors.